discord_bot/discord_bot.py

```diff
@@ -121,6 +128,17 @@ def load_config(config_file_path):
         asyncio.ensure_future(self.heartbeat())
         asyncio.ensure_future(self.handshake())
 
+    async def dispatch_handler(self, message):
+        """
+        Handles all types of DISPATCH events.
+        :param message: dict - the message object.
+        """
+        event = message["t"]
+        if event == Events.READY:
+            self.session_id = message["d"]["session_id"]
+        elif event == Events.MESSAGE_CREATE:
+            await self.respond_message(message)
```

Comment:
Actually, shouldn't we `asyncio.ensure_future`? `await` will block the bot from doing anything until it finishes processing.